src/framework/mod.rs

```diff
@@ -937,16 +940,19 @@ impl ::Framework for BuiltinFramework {
                             if command.use_quotes {
                                 utils::parse_quotes(&content[command_length..])
                             } else {
-                                let delimeter = self.configuration.delimeters
-                                    .iter()
-                                    .find(|d| content.contains(d.as_str()))
-                                    .map(|s| s.as_str())
-                                    .unwrap_or(" ");
-
-                                content
-                                    .split(delimeter)
-                                    .map(|arg| arg.to_owned())
-                                    .collect::<Vec<String>>()
+                                let delimiters = &self.configuration.delimiters;
+                                let regular_expression = delimiters.iter().
```

Comment:
It'd look better (or less awkward) if you put this dot on the same line as `map`.